packages/cli/src/lib/bundler/transforms.ts

```diff
@@ -39,6 +39,7 @@ export const transforms = (options: TransformOptions): Transforms => {
       loader: require.resolve('@sucrase/webpack-loader'),
       options: {
         transforms: ['typescript', 'jsx', ...extraTransforms],
+        disableESTransforms: true,
```

Comment:
This flag sounds so coarse. ALL of them! :$

Reply:
Sucrase is very opinionated, so this fits with that :grin:. It keeps the implementation and maintenance simpler.